src/native/windows/apps/prunsrv/prunsrv.c

```diff
@@ -1887,7 +1932,7 @@ void WINAPI serviceMain(DWORD argc, LPTSTR *argv)
          */
         apxLogWrite(APXLOG_MARK_DEBUG "Waiting for all threads to exit.");
         apxDestroyJvm(INFINITE);
-        reportServiceStatus(SERVICE_STOP_PENDING, NO_ERROR, 0);
+        reportServiceStatus(SERVICE_STOP_PENDING, NO_ERROR, ONE_MINUTE);
```

Comment:
Why hard-coded to a minute?

Reply:
ONE_MINUTE is using in other place and 0 looks wrong, that a hint value for service manager so I don't think that is a problem and there are other location with ONE_MINUTE.